admin.py

```diff
@@ -10,40 +11,44 @@
 from lib import errors
 from lib.utils import clean_json, error, expect_json, success
 
-app = Flask(__name__)
-app.config["SECRET_KEY"] = "ttmadmin"
-CORS(app)
+admin = Flask(__name__)
+admin.config["SECRET_KEY"] = "ttmadmin"
+CORS(admin)
 mdb = MongoClient(db.MONGO_URL).db
 
-@app.route("/auth", methods=["POST"])
-@expect_json(password=str)
-def admin_auth(body):
-    if(body['password'] == "ttmadmin"):
-        return success("Authorized")
-    return error(403, "Incorrect password")

```

Comment:
We do still need an `/auth` endpoint - obviously it won't require auth, since it is the one providing the endpoint; for a dummy implementation it can just return `{"token": "ttmadmin"}` (since that's what's hardcoded into the auth wrapper right now).